integration-tests/lra/src/test/java/org/apache/camel/quarkus/component/lra/it/LraTestResource.java

```diff
@@ -58,7 +58,18 @@ public Map<String, String> start() {
                 lraPort = container.getMappedPort(LRA_PORT);
             }
 
+            // If the test runs with a remote docker, it needs to know the IP of the machine where the test app runs
+            if (System.getenv("DOCKER_HOST") != null) {
+                hostname = System.getProperty("lra.appNode.ip");
```

Comment:
Just curious - what actually uses the `lra.appNode.ip` property? I don't seen any other references to it.

Reply:
the `hostname` variable [is used](https://github.com/avano/camel-quarkus/blob/lra/integration-tests/lra/src/test/java/org/apache/camel/quarkus/component/lra/it/LraTestResource.java#L76) in the `camel.lra.local-participant-url` property

Reply:
Ah I totally missed that, thanks.

So, instead of having to manually pass along a system property, would it be enough to use `DockerClientFactory.instance().dockerHostIpAddress()`?

Reply:
this is not the IP where a docker container runs (rhel sidecar), but the IP where the test application runs (windows). the LRA coordinator runs in the docker container on the remote machine and it needs to be able to invoke the endpoint from the test application

while it is possible to get an IP automatically, it may not work in all envs correctly - for example in our openstack it needs to use an internal openstack ip, as the machines are not reachable using public ips, so I assumed passing the IP manually would be the safest option

Reply:
Ok, lets leave it as-is then. Thanks for the explanation 👍